src/frontend/design-system/components/Form/FormPasswordInput.tsx

```diff
@@ -0,0 +1,25 @@
+import { useLingui } from "@lingui/react";
+import { ISharedFormInput } from "./_types";
+import { generateFormArias, wrapLabelAndError } from "./_wrapForm";
+import { Input } from "./Styles";
+
+interface IFormInput extends ISharedFormInput {
+  type?: "password" | "text";
```

Comment:
This is not needed, you can just pass `ISharedFormInput` as the component props

Reply:
Got it!!